rs/nns/governance/src/governance/tla/mod.rs

```diff
@@ -213,10 +213,25 @@ pub fn check_traces() {
     // improving that later, for now we introduce a hard limit on the state size, and
     // skip checking states larger than the limit
     const STATE_SIZE_LIMIT: u64 = 500;
+    fn is_under_limit(p: &ResolvedStatePair) -> bool {
```

Comment:
This block is not really related to the change, it's just there to print some stats related to the state size filter introduced in #2188